src/api/PersistentSearchApi.js

```diff
@@ -149,8 +150,53 @@ function getPersistentSearches(includeExpired :boolean) :Promise<Object[]> {
     });
 }
 
+/**
+ * `PATCH /persistentsearch/{persistentSearchId}/expiration`
+ *
+ * Updates the expiration date of a particular PersistentSearch
+ *
+ * @static
+ * @memberof lattice.PersistentSearchApi
+ * @param {UUID} persistentSearchId The id of the PersistentSearch to update
+ * @param {string} expiration The new expiration date
+ * @returns {Promise} - a Promise that resolves once the expiration date has been updated
+ *
+ * @example
+ * PersistentSearchApi.updatePersistentSearchExpiration(
+ *   "ec6865e6-e60e-424b-a071-6a9c1603d735",
+ *   "2018-12-22T01:57:02.801Z"
+ * );
+ */
+function updatePersistentSearchExpiration(persistentSearchId :UUID, expiration :string) :Promise<*> {
+
+  let errorMsg = '';
+
+  if (!isValidUUID(persistentSearchId)) {
+    errorMsg = 'invalid parameter: "persistentSearchId" must be a valid UUID';
+    LOG.error(errorMsg, persistentSearchId);
+    return Promise.reject(errorMsg);
+  }
+
+  if (!isNonEmptyString(expiration)) {
+    errorMsg = 'invalid parameter: "expiration" must be a valid string';
+    LOG.error(errorMsg, persistentSearchId);
+    return Promise.reject(errorMsg);
+  }
+
+  return getApiAxiosInstance(PERSISTENT_SEARCH_API)
+    .patch(`/${persistentSearchId}/${EXPIRATION_PATH}`, expiration, {
+      headers: { 'Content-Type': 'application/json' }
+    })
```

Comment:
You shouldn't need the 3rd param with the header... that happens in `newAxiosInstance`.